github/users_projects.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// CreateUserProjectOptions specifies the parameters to the UsersService.CreateProject method.
+type CreateUserProjectOptions struct {
+	// The name of the project. (Required.)
+	Name *string `json:"name"`
```

Comment:
There is no need for the pointer here since this field is required, so please change this from `*string` to `string`.

Also, if you feel like it, it might be nice to move this entire struct to immediately before the `CreateProject` method below so it is more tightly coupled to the location where it is used... but that's not a huge big deal, so I'll leave it up to you.

Reply:
Updated https://github.com/google/go-github/pull/1295/commits/c06ddc162172e25ac117e8e01c5a55f5f86a0c92.

Should I rebase commits to make this clean?

Reply:
No, we avoid force pushes and rebasing in this repo because the final step is always a "Squash and Merge", and it additionally makes code reviews easier because reviewers can see just what changed since the last review. Thanks.